tribits/core/package_arch/TribitsGlobalMacros.cmake

```diff
@@ -1952,6 +1949,20 @@ MACRO(TRIBITS_SETUP_ENV)
     INCLUDE("${PROJECT_COMPILER_CONFIG_FILE}")
   ENDIF()
 
+  # Set up C++ language standard selection.
+  IF (NOT CMAKE_CXX_STANDARD)
+    SET(CMAKE_CXX_STANDARD 11)
```

Comment:
@bradking, just to be clear, but `-std=c++11` will no longer show up in `CMAKE_CXX_FLAGS` that gets exported in the generated `<Package>Config.cmake` or `<Project>Config.cmake` files, right?  I will be to test that for myself.

Reply:
Correct.  I'll update the release notes to mention that.